test/k8sT/manifests/bind/templates/deployment.yaml

```diff
@@ -108,7 +108,7 @@ metadata:
 spec:
   selector:
     zgroup: bind
-  clusterIP: 10.96.0.100
+  {{ if .Values.clusterIP }}clusterIP: {{ .Values.clusterIP }}{{ end }}
```

Comment:
This could be achieved as well with a patch command, similar to https://github.com/cilium/cilium/blob/bd11ea85bd11af298bd47fabf3eb4ecef285a5fe/test/k8sT/external_ips.go#L147-L149

But it's absolutely fine doing it with helm since we need it for `{{ .Values.world1 }}` and `{{ .Values.world2 }}` above

Reply:
Hmm, that is handy. I'm going to leave the template in for now; remote-node on master is making the FQDN tests fail but I'm not 100% sure why yet. There is a chance we will end up without bind entirely but instead patch kube-dns.